src/main/java/com/powsybl/openloadflow/network/impl/LfLegBranch.java

```diff
@@ -74,16 +78,43 @@ public static LfLegBranch create(LfBus bus1, LfBus bus0, ThreeWindingsTransforme
                         .setA1(a1));
             }
             piModel = new PiModelArray(models, ptc.getLowTapPosition(), ptc.getTapPosition());
+        }
 
-        } else {
+        RatioTapChanger rtc = leg.getRatioTapChanger();
```

Comment:
Same here

Reply:
I have tried to make some improvements and I have created a test but... I fall in powsybl-core exceptions `com.powsybl.iidm.network.ValidationException: 2 windings transformer 'T2wT': Only one regulating control enabled is allowed`